third_party/amd/lib/TritonAMDGPUTransforms/ReorderInstructions.cpp

```diff
@@ -182,7 +181,8 @@ static void moveUpGlobalLoad(triton::FuncOp funcOp) {
       // Only move ops residing in the same block.
       return defBlock == block;
     };
-    mlir::getBackwardSlice(op, &backwardSet, options);
+    mlir::getBackwardSlice(static_cast<mlir::Operation *>(op), &backwardSet,
```

Comment:
Do we need the extra `static_cast` here?

Reply:
Looks like we do. Or it cant choose between `Operation *` and `Value`

Reply:
Seems like `*op` + implicit conversion doesn't work. But `op.getOperation()` works.